lib/manager/npm/post-update/index.ts

```diff
@@ -129,9 +130,10 @@ export async function writeExistingFiles(
     logger.debug('Removing ignored .npmrc file before artifact generation');
     await remove(npmrcFile);
   }
-  if (config.yarnrc) {
+  if (is.string(config.yarnrc)) {
     logger.debug(`Writing repo .yarnrc (${config.localDir})`);
-    await outputFile(upath.join(config.localDir, '.yarnrc'), config.yarnrc);
+    const yarnrcFileName = upath.join(config.localDir, '.yarnrc');
```

Comment:
I think this will already be done by outputFile?

Reply:
actually this change isn't necessary, i will revert it. We could refactor to use outputLocalFile but I don't want to complicated it right now